index.html

```diff
@@ -12,30 +13,33 @@
     <!-- This is the main content of the page -->
     <main>
       <!-- First section with header and paragraph -->
-      <section class="section1">
-        <h1>Testing Header One</h1>
+      <section>
+        <div id="header">
+          <h1>Testing Header one</h1>
+        </div>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias
-          sed, accusantium dolorum a est distinctio non itaque ipsam excepturi
-          harum nihil iure? Eum perspiciatis, animi ut ipsa facilis fugiat
-          vitae.
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias sed,
+          accusantium dolorum a est distinctio non itaque ipsam excepturi harum
+          nihil iure? Eum perspiciatis, animi ut ipsa facilis fugiat vitae.
         </p>
       </section>
-      
-      <!-- Second section with header and paragraph -->
```

Comment:
@StanleyAnyas I think u hv deleted this comment by error

Reply:
Attend to this boss

Reply:
What do i have to do

Reply:
copy the comment and paste it right back into the codebase where it's meant to be